ember-app/app/models/repo.js

```diff
@@ -19,6 +20,7 @@ var Repo = Ember.Object.extend(Serializable,{
   fetchBoard: function(linkedBoards){
     if(this._board) {return this._board;}
     var self = this;
+    Health.fetch(self);
```

Comment:
Hmm might have brought that file back from a merge or something. 

Don't worry about it, it's not being used